package.json

```diff
@@ -35,7 +35,7 @@
     "react-native-maps": "0.26.1",
     "react-native-paper": "^3.2.1",
     "react-native-picker-select": "^6.3.3",
-    "react-native-qrcode-svg": "^6.0.1",
+    "react-native-qrcode-svg": "^6.0.6",
```

Comment:
Did this change for a reason? If not, make sure to run npm install after reverting this file in order to ensure the package-lock.json file matches.